README.md

```diff
@@ -18,6 +18,10 @@ RCOS offers a huge community of open source developers skilled in nearly any pla
 
 RCOS is offered for credit at RPI and fulfills a 4000 level technical elective.
 
+# How to join RCOS
+
+If your an RPI student, hop in on our first meeting of the semester or join [our slack](https://rcos.slack.com) and message a faculty coordinator (@mskmoorthy or @turnew2). If you're looking to get involved with RCOS as a member outside of RPI, email a [coordinator](mailto:richiejoeyoung@gmail.com).
```

Comment:
yea that would be a really good idea, and yea I'll rebase his email out- we've added emails to the README in the past but I guess it might be a bad idea as a general practice.

The reason why external members have to email a coordinator as of now is because our slack is restricted, I'm very open to the idea of opening our slack to all domains (not just @rpi.edu domains).

Ping to @kmcnellis